composer/trainer/trainer.py

```diff
@@ -944,6 +955,13 @@ def __init__(
                                               strict_model_weights=load_strict_model_weights,
                                               chunk_size=load_chunk_size,
                                               progress_bar=load_progress_bar)
+            # If using autoresume, use loaded run_name
+            if autoresume and run_name is not None:
```

Comment:
For now, can we always override the run name to whatever was used for `Event.INIT`?

e.g. `self.state.run_name = generated_run_name`